src/Umbraco.Web.UI.Client/src/views/components/umb-node-preview.html

```diff
@@ -48,15 +48,15 @@
         </button>
 
         <!-- If removeUrl has a value we render a link otherwise a button -->
-        <a class="umb-node-preview__action umb-node-preview__action--red"
+        <a class="umb-node-preview__action"
             ng-href="{{changeUrl}}"
             ng-if="allowChange && changeUrl"
             ng-click="onChange()">
             <localize key="general_change">Change</localize><span class="sr-only">&nbsp;{{name}}</span>
         </a>
     
         <button type="button"
-                class="umb-node-preview__action umb-node-preview__action--red"
+                class="umb-node-preview__action"
```

Comment:
I think only "Remove" button should have red class. I feel the Change button should be similar to Edit button. I fact we could probably just have used same button as edit/change is kind of synonyms
https://www.merriam-webster.com/thesaurus/edit

and instead options to customize the link/button text.

but since it has already been added, we can adjust it a bit.